pkg/vault/test/foundry/fork/BoostedPoolWithEmptyBuffer.t.sol

```diff
@@ -24,7 +24,8 @@ import { RouterCommon } from "../../../contracts/RouterCommon.sol";
 import { BaseVaultTest } from "../utils/BaseVaultTest.sol";
 import { ERC4626RateProvider } from "../../../contracts/test/ERC4626RateProvider.sol";
 
-contract BoostedPoolWithEmptyBufferTest is BaseVaultTest {
+// TODO: rename file. Do later so that we can see the diff.
+contract YieldBearingPoolWithEmptyBufferTest is BaseVaultTest {
```

Comment:
Maybe we need to rename the file to YieldBearingPoolWithEmptyBuffer?

Reply:
Yes; shouldn't have Boosted anywhere in the code, including filenames.